### PR TITLE
Add categorical colors to color maps

### LINKED ISSUE #607: For many component data, generate distinguishable black -> colorX per-component colors
Per discussion in #605

### PR DESCRIPTION
**Pull request by PaulHax**
Selecting a categorical color picks one based on component index.

Depends on https://github.com/InsightSoftwareConsortium/itk-viewer-color-maps/pull/1

closes #607

### CODE REVIEW

**Comment by thewtex** [comment]
> Depends on InsightSoftwareConsortium/itk-viewer-color-maps#1

Pushed to npmjs as itk-viewer-color-maps@1.1.0

File: src/UI/reference-ui/src/createColorMapIconSelector.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+import { ColorMapIcons, CategoricalColorIcons } from 'itk-viewer-color-maps'
+
+const REMOVED_COLOR_MAPS = [
+  'Purples',
+  'Oranges',
+  'PuBu',
+  'BuPu',
+  'BuGn',
+  'GnBu',
+]
```
Comment (thewtex) [inline src/UI/reference-ui/src/createColorMapIconSelector.js:11]: Maybe we should just remove them from itk-viewer-color-maps? What do you think?

File: src/Rendering/VTKJS/Images/applyLookupTable.js [hunk 2]
```diff
@@ -27,11 +27,6 @@ function applyLookupTable(context, event) {
 
   const uniqueLabels = Array.from(actorContext.uniqueLabels)
 
-  colorTransferFunction.setMappingRange(
-    uniqueLabels[0],
-    uniqueLabels[uniqueLabels.length - 1]
-  )
-
```
Comment (thewtex) [inline src/Rendering/VTKJS/Images/applyLookupTable.js:34]: Why is this removed?

**Review by thewtex — COMMENTED** [review]
:rainbow: :sparkle:

File: src/Rendering/VTKJS/Images/applyLookupTable.js (hunk 2 above)
Comment (PaulHax) [reply src/Rendering/VTKJS/Images/applyLookupTable.js:34]: `setMappingRange` is called in call to `applyCategoricalColorToColorTransferFunction` on next line.

File: src/UI/reference-ui/src/createColorMapIconSelector.js (hunk 1 above)
Comment (PaulHax) [reply src/UI/reference-ui/src/createColorMapIconSelector.js:11]: 👍️ Lighten the load a bit.

**Review by thewtex — APPROVED** [review]
